app/react/Settings/components/CollectionSettings.tsx

```diff
@@ -53,6 +53,7 @@ const CollectionSettings = ({
   register('mapStartingPoint');
   register('cookiepolicy');
   register('newNameGeneration');
+  register('toggleOCRButton');
```

Comment:
This is not toggling on a button, it's toggling on an entire feature.  Is this the name in the collection as well?  Please, rename it to: "toggleOCRService".

Reply:
Agreed. We are changing it to "ocrServiceEnabled". We think it's more descriptive than an "action" phrase.